hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -798,9 +798,19 @@
 
   /**
    * Parameter name for client pause value for special case such as call queue too big, etc.
+   * @deprecated Since 2.6.0, will be removed in 4.0.0. Please use
+   *    hbase.client.pause.server.overloaded instead.
    */
+  @Deprecated
```

Comment:
You also should add a mapping between these values via `Configuration#addDeprecation()`. I'm actually shocked that I cannot find any uses of this method in the HBase codebase. I was hoping there would be an obvious place to add it. Maybe someone else can review and suggest where. Basically, I think we need a static code block someplace where the deprecation can be registered once. Perhaps in the static initializer of the class where it's used, `AsyncConnectionConfiguration` ?

Reply:
I ended up doing it in ConnectionConfiguration, where I define the replacement.